wiki/Beatmap/Difficulty/en.md

```diff
@@ -1,15 +1,14 @@
 ---
 tags:
-  - difficulty
   - difficulty name
```

Comment:
probably add the plural noun in tags now

```suggestion
  - difficulties
  - difficulty name
```

Reply:
makes sense